mpc/share-conversion/mpc-share-conversion/src/converter.rs

```diff
@@ -0,0 +1,278 @@
+use std::sync::{Arc, Weak};
+
+use async_trait::async_trait;
+use mpc_share_conversion_core::{Field, Share};
+
+use crate::{
+    AdditiveToMultiplicative, GilboaReceiver, GilboaSender, MultiplicativeToAdditive,
+    OTReceiveElement, OTSendElement, ReceiverConfig, SenderConfig, ShareConversionChannel,
+    ShareConversionError,
+};
+
+/// The share conversion sender
+pub struct ConverterSender<F: Field, OT> {
+    ot: OT,
+    sender: Option<Arc<GilboaSender<F>>>,
```

Comment:
If I remember correctly, only one version was called "Gilboa". I think it was M2A from this paper on page 6 https://link.springer.com/content/pdf/10.1007/3-540-48405-1_8.pdf